apps/service_acquire/lib/acquire/query/where/temporal.ex

```diff
@@ -28,6 +29,12 @@ defmodule Acquire.Query.Where.Temporal do
     |> and_clause()
   end
 
+  defp dictionary(dataset_id, subset_id) do
+    with {:ok, nil} <- identifier(dataset_id, subset_id) |> Acquire.ViewState.Fields.get() do
+      Ok.error("dictionary not found for #{dataset_id} #{subset_id}")
+    end
+  end
```

Comment:
is there a central way to share this boiler plate function with the `Where.Bbox` module?